src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -206,7 +334,23 @@ public static void setMythicVariableMap(Entity entity, Map<String, Variable> map
         registry.putAll(map);
     }
 
+    public static ItemExecutor getItemManager() {
+        return MythicBukkit.inst().getItemManager();
+    }
+
+    public static MythicItem getMythicItem(String name) {
+        return MythicBukkit.inst().getItemManager().getItem(name).orElse(null);
+    }
+
+    public static Collection<String> getSkillNames() {
+        return MythicBukkit.inst().getSkillManager().getSkillNames();
+    }
+
+    public static SkillExecutor getSkillManager() {
+        return MythicBukkit.inst().getSkillManager();
+    }
+
     public static boolean skillExists(String name) {
```

Comment:
This method was used by `MythicSkillCommand` but is now unused, can probably be removed

Reply:
Still relevant